app/components/pdf-document.js

```diff
@@ -0,0 +1,264 @@
+/* globals PDFJS Promise */
```

Comment:
the lint list above:

````
/* globals PDFJS Promise */
````

... should be changed to:

````
/* globals PDFJS Promise window */
````

see: http://jshint.com/docs/options/#globals